src/constant/constant.ts

```diff
@@ -23,6 +23,7 @@ export enum UserStatus {
   Active = "active",
   Inactive = "inactive",
   Invited = "invited",
+  Deactivate = "deactivate",
```

Comment:
@Mubashar1009 We should also consider the potential side effects of adding a new key entries in existing enum.

Reply:
This should be called `deactivated`

Reply:
Why do we have User status as both enum and type. This is confusing.